src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,44 @@
+package unit.org.pih.warehouse.api
+
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import org.testng.Assert
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.*;
```

Comment:
I would say importing everything from a package is not a good practice. We should import only what is needed. (Remove `*` and replace it with imports that are needed)

Reply:
ok i will make this changes